wp/lib/bap_wp/src/environment.ml

```diff
@@ -228,41 +222,54 @@ type unfold_depth = int Unfold_depth.t
      recursively call [Precondition.visit_graph]. Because this function is defined in another
      (later) module, we use open recursion via the [wp_rec_call] function reference. *)
 let rec loop_unfold (num_unroll : int) (depth : unfold_depth) : loop_handler =
```

Comment:
I have no idea what this change is changing.

Reply:
There's basically no change. I just took the handler out of the record to match the change I made to the `loop_handler` type.